### PR TITLE
Update Russian translation

### PR DESCRIPTION
**Pull request by Districh-ru**

### CODE REVIEW

File: files/lang/ru.po [hunk 1]
```diff
@@ -3856,155 +3848,146 @@ msgstr "Характер:"
...
-msgstr ""
+msgstr "Разрешить строительство замка в данном городе."
 
 msgid "Toggle the use of default buildings. Custom buildings will be reset!"
-msgstr ""
+msgstr "Использовать здания по умолчанию. Постройки будут сброшены!"
 
 msgid "Toggle building construction restriction mode."
-msgstr ""
+msgstr "Переключит режим запрета постройки зданий."
```
Comment (ihhub) [inline files/lang/ru.po:3879]: Missing character **ь**: `Переключить`.

File: files/lang/ru.po [hunk 2]
```diff
@@ -3856,155 +3848,146 @@ msgstr "Характер:"
...
+msgstr "Использовать армию защитников по умолчанию."
 
 msgid "Set custom Castle Army. Right-click to reset unit."
-msgstr ""
+msgstr "Установить армию замка. Правый клик - сброс отряда."
 
-#, fuzzy
 msgid "Message Text:"
-msgstr "Информационная строка"
+msgstr "Текс сообщения:"
```
Comment (ihhub) [inline files/lang/ru.po:3891]: `Текс` --> `Текст`.

File: files/lang/ru.po [hunk 3]
```diff
@@ -3856,155 +3848,146 @@ msgstr "Характер:"
...
+msgstr "Армия по умолч."
 
-#, fuzzy
 msgid "Castle Army"
-msgstr "Замок"
+msgstr "Армия замка"
 
 msgid "Click to change the Castle name. Right-click to reset to default."
-msgstr ""
+msgstr "Щёлкните чтобы изменить название замка. Правый клик - сбросить."
```
Comment (ihhub) [inline files/lang/ru.po:3867]: I think it should be a comma after the first word.

File: files/lang/ru.po [hunk 4]
```diff
@@ -3856,155 +3848,146 @@ msgstr "Характер:"
...
 
 msgid "Allow %{color} human player to get event"
-msgstr ""
+msgstr "Позволить событие для игрока-человека цвета: %{color}"
 
 msgid ""
 "If this checkbox is checked, this event will trigger for the %{color} player "
 "if they are controlled by a human."
 msgstr ""
+"Если этот флажок установлен, это событие произойдёт для игрока-человека если "
```
Comment (ihhub) [inline files/lang/ru.po:3912]: A missing comma before the second `если` word.

File: files/lang/ru.po [hunk 5]
```diff
@@ -3856,155 +3848,146 @@ msgstr "Характер:"
...
+"его цвет: %{color}."
```
Comment (ihhub) [inline files/lang/ru.po:3913]: I don't think that **:** is needed here.

File: files/lang/ru.po [hunk 6]
```diff
@@ -4383,87 +4355,70 @@ msgstr "Нет доступных карт!"
...
 
-#, fuzzy
 msgid "Map Name"
-msgstr "Название файла карты"
+msgstr "Название карты"
 
-#, fuzzy
 msgid "Click to change the description of the current map."
-msgstr "Расширенное описание выбранной карты."
+msgstr "Нажмите чтобы изменить описание текущей карты."
```
Comment (ihhub) [inline files/lang/ru.po:4405]: Missing comma after the first word.

File: files/lang/ru.po [hunk 7]
```diff
@@ -7833,26 +7783,26 @@ msgid "Set army combat formation to 'Grouped'"
...
+msgstr "Нажмите для изменения имени героя. Правый клик - сбросить."
 
 msgid "Hero is in patrol mode in %{tiles} tiles radius. Click to disable it."
 msgstr ""
+"Герой патрулирует область радиусом в %{tiles} клеток. Нажмите для отключения."
 
-#, fuzzy
 msgid "Click to enable hero's patrol mode."
-msgstr "Нажмите сюда чтобы выбрать этот цвет."
+msgstr "Нажмите чтобы установить героя в режим патруля."
```
Comment (ihhub) [inline files/lang/ru.po:7805]: A missing comma after the first word.

**Review by ihhub — CHANGES_REQUESTED** [review]
Hi @Districh-ru , I left several comments here. Could you please take a look once you have time?

**Review by ihhub — APPROVED** [review]

File: files/lang/ru.po [hunk 8]
```diff
@@ -5227,11 +5180,12 @@ msgstr "' языке, но текущий язык игры - "
 msgid "Hot Seat"
 msgstr "За одним компьютером"
 
-#, fuzzy
 msgid ""
 "Play a Hot Seat game, where 2 to 6 players play around the same computer, "
 "switching into the 'Hot Seat' when it is their turn."
-msgstr "Позволяет нескольким игрокам играть за одним компьютером."
+msgstr ""
+"Сыграть в игру в режиме \\\"Горячий стул\\\", когда от 2 до 6 игроков играют "
```
Comment (ihhub) [inline files/lang/ru.po:5187]: Could you please explain why we have triple \\\ here?

Comment (Districh-ru) [reply files/lang/ru.po]: Ow, it's a copy from source and paste to Poedit typo. I've fixed it.

**Comment by ihhub** [comment]
@Districh-ru , thank you very much for the update!

**Pull request merged**